### ISSUE TITLE
Can't install any CIA via FBI.

### ISSUE THREAD

**Issue by DanielBrenoM** [Steveice10/FBI#435]
Hi. I'm trying to install cias on my o3DsXL but every time I try I get error:
```
Failed to install CIA file.
Result code: 0xD8E0806A
Level: Permanent (27)
Summary: Invalid argument (7)
Module: AM (32)
Desc: Cert signature or hash check failed (106) 
```
I've alerady searched the internet, and tried to install the cias by injecting FBI onto H&S, by going into HBC, and through Rosalina in Download Play but, the same error appears, I've managed to install HBC and DSP1, but I can't install FBI itself nor any other CIA, I've tried downloading the latest CIAs but none of it fixed the problem, I really don't know what to do. Thanks in advance and sorry if I shouldn't be asking this here.

**Comment by Steveice10** [comment]
For some reason or another, the AM service is rejecting the CIA files. I assume you're already in a CFW since you mentioned Rosalina, so I would check the validity of the other CIAs you're failing to install. It might also help to run a check on your SD card's filesystem.

**Comment by DanielBrenoM** [comment]
Hi, thank you for you response.
Well, I've already tried installing from TitleDB, and through QR Codes, but the same error shows up. 
Like I said, I've only managed to install HBC and DSP1, but the other CIAs I have fail, I've tried downloading every Cia from every github page to see if it was that, but the same thing happend, the only thing I haven't done was check my SD card's filesystem. I'll do that and see if it is the problem.
